test/code_generator_test.dart

```diff
@@ -436,7 +440,7 @@ final ffi.DynamicLibrary _dylib;
 /// The symbols are looked up in [dynamicLibrary].
 Bindings(ffi.DynamicLibrary dynamicLibrary): _dylib = dynamicLibrary;
 
-late final ffi.Pointer<int> _test1 = _dylib.lookup<ffi.Int32>('test1');
```

Comment:
How did this not fail before?

Reply:
In the last PR, Tim had updated this manually after generating the result and since the code_generator_tests compare the raw output by ffigen, `dartfmt` didn't run on this.

Perhaps I should update the code_generator_tests to completely generate bindings (including running dartfmt) and then compare the results? and should I do this in a new PR or this one itself?

Reply:
Separate PR is fine.

Kinda nasty that dartfmt doesn't run on some things, otherwise we could just add tests that check that everything is formatted. Can you take a look at trying to run dartfmt on all Dart files in the repo on the CI and error out if one of them is not formatted as expected or fails to format. Maybe we can have a list of files that we should skip when checking so we can keep the CI green.